mussidae/time_range_tools/false_positives.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import absolute_import
+import os
+import numpy as np
+from numpy.lib.recfunctions import append_fields
+import dateutil.parser
+import logging
+import csv
+from cStringIO import StringIO
+import mussidae.time_range_tools as trtools
+
+import bqtools
+
+proj_id = 'world-fishing-827'
```

Comment:
We should be extracting this 'world-fishing-827' somewhere common to Scala + python. I'll take a go at that for the ticket re-arranging the GS output paths we use. In the meantime any chance there could be a common python module that we all use that has this project name, and possibly a shared root path for temp data? I suggest putting it under `gs://world-fishing-827-dev-ttl30d`.